examples/music-v2.py

```diff
@@ -33,11 +33,12 @@ def __unload(self):
 
         # Clears the players from Lavalink's internal cache
         self.bot.lavalink.players.clear()
-        self.bot.lavalink.unregister_hook(self._track_hook)
 
     async def _track_hook(self, event):
         if isinstance(event, lavalink.Events.StatsUpdateEvent):

```

Comment:
Maybe instead of testing for different events do this: `if not hasattr(event, "player"): return`?
*Applies to all cogs*